kmod/kcompat.h

```diff
@@ -4690,6 +4690,12 @@ static inline bool page_is_pfmemalloc(struct page __maybe_unused *page)
 #define HAVE_NDO_DFLT_BRIDGE_GETLINK_VLAN_SUPPORT
 #endif /* 4.2.0 */
 
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,0,0))
+#define HAVE_NETLINK_EXT_ACK
+#endif
+
+
+
```

Comment:
Some extra spaces here. And maybe add this code after the checks for kernel 4.20? Could make skimming the code easier, but 4.19 and 4.20 are inverted already...